SwiftUIBasic/SwiftUIBasic/Authentication/ViewModel/AuthenticationViewModel.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AuthenticationViewModel.swift
+//  SwiftUIBasic
+//
+//  Created by Jaesung Lee on 2021/06/14.
+//
+
+import Combine
+import Commons
+import SendBirdCalls
+
+class AuthenticationViewModel: ObservableObject {
+    @Published var userID: String = ""
+    @Published var accessToken: String = ""
```

Comment:
This doesn't seem to be used. Just assign nil every time?

Reply:
I added new textfield for access token in `AuthenticationView`